hvpy/api_groups/jpeg2000/get_jp2_image.py

```diff
@@ -0,0 +1,11 @@
+from typing import Optional
+from datetime import datetime
+
+from hvpy.io import HvpyParameters
+
+
+class getJP2ImageInputParameters(HvpyParameters):
+    date: datetime
+    sourceId: int
+    jpip: Optional[bool] = False
+    Json: Optional[bool] = False
```

Comment:
This is where `get_output_type()` becomes useful.

Per the [API details](https://api.helioviewer.org/docs/v2/api/api_groups/jpeg2000.html#getjp2image)
> If request parameter jpip is set to true (and the json parameter is omitted or set to false) the reponse is a plain text string.

To handle this you have to override `get_output_type()` to check the following conditions:

if jpip = false and json = false, then output will be Raw

if jpip = true and json = false, then output will be String

if jpip = true and json = true, then output will be JSON